### PR TITLE
Change component options boolean props wording

### PR DESCRIPTION
**Pull request by Jivanvl**
This pull requests continues the work from: https://github.com/ecomfe/echarts-doc/pull/38

This is for the component options, I'll be creating multiple branches for the different API options

### CODE REVIEW

File: en/option/component/title.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ In ECharts 2.x, a single instance of ECharts could contains one title component
 
 ## show(boolean) = true
 
-It specifies whether to show the title component.
+Change this to `false` to prevent the title from showing
```
Comment (Ovilia) [inline en/option/component/title.md]: I guess it's better to use the same expression as others, probably `Set this ...`?

Comment (Jivanvl) [reply en/option/component/title.md]: Oh yes! let's keep it consistent

**Comment by Jivanvl** [comment]
@Ovilia Made the requested changes, thanks!

**Comment by Ovilia** [comment]
@Jivanvl Thanks!

**Pull request merged**